en_US/advanced/cluster.md

```diff
@@ -58,6 +58,47 @@ Cookies are used for interconnection authentication between Erlang nodes. A cook
 
 See <http://erlang.org/doc/reference_manual/distributed.html> for details.
 
+### Using TLS for backplane connections
+It is possible to enable TLS encryption for the backplane connections. It comes at the cost of increased CPU load, though.
+
+1. Generate a CA-signed certificate using `openssl` tool:
+  ```
+  # Create self-signed root CA:
+  openssl req -nodes -x509 -sha256 -days 1825 -newkey rsa:2048 -keyout rootCA.key -out rootCA.crt -subj "/O=LocalOrg/CN=LocalOrg-Root-CA"
+  # Create a private key:
+  openssl genrsa -out domain.key 2048
+  # Create openssl extfile:
+  cat <<EOF > domain.ext
+  authorityKeyIdentifier=keyid,issuer
+  basicConstraints=CA:FALSE
+  subjectAltName = @alt_names
+  [alt_names]
+  DNS.1 = backplane
+  EOF
+  # Create a CSR:
+  openssl req -key domain.key -new -out domain.csr -subj "/O=LocalOrg"
+  # Sign the CSR with the Root CA:
+  openssl x509 -req -CA rootCA.crt -CAkey rootCA.key -in domain.csr -out domain.crt -days 365 -CAcreateserial -extfile domain.ext
+  ```
```

Comment:
maybe split this to two steps
step 1. generate a root ca (key and self-signed cert)
step 2. generate server key and issue a certificate from the ca.

repeat step 2 for other nodes in the cluster. (i.e. they should share the same CA cert)

Reply:
The nodes can all use the same cert.

Reply:
ok. can we add more description about this then?
e.g.

> When clustered, the nodes may share the same set of key and certificate files, otherwise they must at leat share (trust) the same (collection of) root CA.

Reply:
Done and done.